src/js/color_change.js

```diff
@@ -255,64 +255,52 @@ $(function() {
 
     $('#courseListTable th')
         .not(':last')
-        .click(function() {
-            var $this = $(this);
-            var isSorted = false;
-
-            // check if the column is already sorted
-            if ($this.hasClass('sorted')) {
-                isSorted = true;
+        .on('click', function() {
+            var isAscending = false,
+                isDescending = false;
+            var $items = retrieveColumnItems($(this));
+            console.log($items);
```

Comment:
let's remove this log

Reply:
Oh right, sorry added it while debugging.